modules/opengl/src/debugmessages.cpp

```diff
@@ -192,9 +192,11 @@ void handleOpenGLDebugMode(Canvas::ContextID context) {
     auto mode = openglSettings->debugMessages_.getSelectedValue();
     auto severity = openglSettings->debugSeverity_.getSelectedValue();
     setOpenGLDebugMode(mode, severity);
-    logDebugMode(mode, severity, context);
+    if (mode != debug::Mode::Off) {
+        logDebugMode(mode, severity, context);
```

Comment:
Should not logDebugMode handle off mode?

Reply:
This is for the initial message when a canvas is created. The reasoning here is that you don't want to see this message every time a canvas is created (applies to canvas processor, port inspectors, etc.)

Reply:
On the other hand, you might want to see the status change from "debug" to "off".